### PR TITLE
Use parchment_delta instead of quill_delta

### PR DESCRIPTION
**Pull request by Amir-P**
- Use parchment_delta instead of quill_delta
- Export parchment_delta from parchment
- Follow https://dart.dev/guides/libraries/create-packages#importing-library-files

### CODE REVIEW

**Comment by Amir-P** [comment]
I'm a bit hesitated about exporting `parchment_delta`. What do you think? @amantoux

File: packages/fleather/lib/src/widgets/theme.dart [hunk 1]
```diff
@@ -2,7 +2,6 @@
 // for details. All rights reserved. Use of this source code is governed by a
```
Comment (amantoux) [inline packages/fleather/lib/src/widgets/theme.dart:2]: We can remove the copyright stuff I think

File: packages/fleather/lib/util.dart [hunk 2]
```diff
@@ -9,7 +9,6 @@ import 'dart:math' as math;
 import 'dart:ui';
```
Comment (amantoux) [inline packages/fleather/lib/util.dart:9]: We can remove copyright stuff also

File: packages/fleather/test/testing.dart [hunk 3]
```diff
@@ -8,7 +8,6 @@ import 'package:fleather/src/widgets/text_selection.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_test/flutter_test.dart';
-import 'package:quill_delta/quill_delta.dart';
```
Comment (amantoux) [inline packages/fleather/test/testing.dart:11]: Copyright to remove?

File: packages/fleather/test/util_test.dart [hunk 4]
```diff
@@ -2,8 +2,8 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
```
Comment (amantoux) [inline packages/fleather/test/util_test.dart:3]: copyright to remove?

File: packages/fleather/test/widgets/controller_test.dart [hunk 5]
```diff
@@ -5,7 +5,6 @@ import 'package:fake_async/fake_async.dart';
 import 'package:fleather/fleather.dart';
```
Comment (amantoux) [inline packages/fleather/test/widgets/controller_test.dart:5]: Copyright to remove?

File: packages/parchment/lib/parchment.dart [hunk 6]
```diff
@@ -5,6 +5,8 @@
 /// Rich text document model for Fleather editor.
```
Comment (amantoux) [inline packages/parchment/lib/parchment.dart:5]: Copyright to remove?

File: packages/parchment/lib/src/codecs/markdown.dart [hunk 7]
```diff
@@ -4,7 +4,7 @@
```
Comment (amantoux) [inline packages/parchment/lib/src/codecs/markdown.dart:4]: copyright to remove?

File: packages/parchment/lib/src/document.dart [hunk 8]
```diff
@@ -3,7 +3,7 @@
 // BSD-style license that can be found in the LICENSE file.
 import 'dart:async';
```
Comment (amantoux) [inline packages/parchment/lib/src/document.dart:4]: Copyright to remove?

File: packages/parchment/lib/src/document/block.dart [hunk 9]
```diff
@@ -1,7 +1,8 @@
 // Copyright (c) 2018, the Fleather project authors.  Please see the AUTHORS file
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
-import 'package:quill_delta/quill_delta.dart';
```
Comment (amantoux) [inline packages/parchment/lib/src/document/block.dart:4]: copyright to remove?

**Comment by amantoux** [comment]
> I'm a bit hesitated about exporting `parchment_delta`. What do you think? @amantoux

I find it appealing but It see deltas a pure formalism that should always be backward compatible. Therefore binding Parchment clients to the same delta version might be too much of a constraint, non?

**Review by amantoux — CHANGES_REQUESTED** [review]
Thanks for remembering to use parchment_quill I had forgotten about this :)
I left you a reply with respect to exporting parchment_quill.

Also, do you mind removing old copyrights from files? (I started noting down where they appeared but they are all over Parchment files

**Review by amantoux — APPROVED** [review]
LGTM

**Pull request merged**